### PR TITLE
Fix/separate big play button

### LINKED ISSUE #7: Extend the BigPlayButton class with a 360BigPlayButton class
Right now whenever this plugin is included it always overwrites the default style of the big play button so that it is a 360 video button. (See css). We should probably extend the BigPlayButton class in video.js and only replace the BigPlayButton when the video that is playing is know to be 360. We should then only style that button rather than the big play button.

### PR DESCRIPTION
**Pull request by BrandonOCasey**
Fixes #7 
Based off of #10

### CODE REVIEW

File: src/plugin.js [hunk 1]
```diff
@@ -351,6 +378,18 @@ const initPlugin = function(player, options) {
       return;
     }
 
+    player.removeChild('BigPlayButton');
```
Comment (gkatsev) [inline src/plugin.js]: we should null (or set to the vr bpb) out `player.bigPlayButton` as well and restore it on `reset`

Comment (BrandonOCasey) [inline src/plugin.js]: 👍

File: src/plugin.js [hunk 2]
```diff
@@ -44,25 +48,24 @@ const errors = {
...
-  let movieMaterial;
-  let movieGeometry;
-  let movieScreen;
-  let controls3d;
-  let scene;
-  let cameraVector;
+  const videoEl = player.el().getElementsByTagName('video')[0];
+  const container = player.el();
+  const defaultProjection = settings.projection;
+  const oldBigPlayButton = player.BigPlayButton;
```
Comment (gkatsev) [inline src/plugin.js]: this should be `bigPlayButton`.

Comment (gkatsev) [inline src/plugin.js]: Also, we probably don't need to store the old one. If it wasn't removed, we can just get the reference via `getChild` and then set it `reset`. One less thing to track ourselves here.

Comment (BrandonOCasey) [inline src/plugin.js]: [We do use `removeChild` on BigPlayButton](https://github.com/videojs/videojs-vr/pull/11/files#diff-a4f485aa834ca5b3c24819fceb39c56fR384)

Comment (gkatsev) [inline src/plugin.js]: Bug given that we add a new one in `reset`, we don't need to store a reference here.

**Pull request merged**